src/components/Spinner/Spinner.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
It would be better to leave the Spinner files out of this branch since they belong to the next ticket. It's OK for now, just good to keep in mind to separate the code according to the acceptance criteria in the ticket